index.js

```diff
@@ -213,7 +213,7 @@ function toBuffer (v) {
       }
     } else if (typeof v === 'number') {
       if (!v) {
-        v = Buffer.from([])
+        v = Buffer.alloc(1)
```

Comment:
Either specify the value explicitly (such as `from([0])`) or use `allocSafe` because that initialises with zero.

Reply:
got it.